src/components/ToggleButtonsStaysExperiences/ToggleButtonsStaysExperiences.module.css

```diff
@@ -9,7 +11,7 @@
     border-radius: 24px;
     transition: background-color 0.3s, color 0.3s, font-weight 0.3s;
     margin-right: none;
-    font-size: 17px;
+    /* font-size: 17px; */
```

Comment:
If not required this line can be deleted without commenting.

Reply:
![image](https://github.com/user-attachments/assets/1005632b-5534-4751-8b66-e6b1068e2611)
When selected 'stay' or 'experience' it has a border. It can be fixed setting outline to 'none'

&:focus {
        outline: none;
    }